src/ci/docker/host-x86_64/mingw-check/Dockerfile

```diff
@@ -39,7 +41,7 @@ COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/
 ENV RUN_CHECK_WITH_PARALLEL_QUERIES 1
 ENV SCRIPT python3 ../x.py --stage 2 test src/tools/expand-yaml-anchors && \
            python3 ../x.py check --target=i686-pc-windows-gnu --host=i686-pc-windows-gnu --all-targets && \
-           python3 ../x.py build --stage 0 src/tools/build-manifest && \
+           python3 ../x.py build --stage 0 src/tools/build-manifest library/std && \
```

Comment:
Not sure whether we need to invoke building `library/std` explicitly.

Reply:
You don't have to :) it's already built automatically here: https://github.com/rust-lang/rust/blob/7d3457bb1220df1afac72eab41806c82cb67d250/src/ci/docker/host-x86_64/mingw-check/Dockerfile#L46